src/app/containers/AudioVideo/Canonical.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string, bool } from 'prop-types';
+
+const Canonical = ({ id, portrait }) => {
+  const position = portrait ? 'position: absolute;' : '';
+
+  const PortraitParent = styled.div`
+    padding-bottom: 178.78%;
+    position: relative;
+  `;
+
+  const MediaPlayerContainer = styled.div`
+    height: 100%;
+    width: 100%;
+    ${position}
+  `;
+
+  if (portrait) {
```

Comment:
You could do something like this for a single return? Not fussed either way

```js
const Parent = portrait ? PortraitParent : Fragment;

return (
  <Parent>
    <MediaPlayerContainer id={id} key={id} />
  </Parent>
);
```

Reply:
I think it's slightly more readable as-is.